### DISCUSSION TITLE
The Shape of Numbers (4,5,6)

### DISCUSSION THREAD

**Discussion by mkhanjani** [rastikerdar/vazir-font#193] [General]
Why you use none-Persian style for numbers like 4,5,6?! 
Please compare these numbers in "B Nazanin" font and "Vazir" or "Sahel" fonts.

**Comment by rastikerdar** [comment]
I think your keyboard layout is not standard Persian!
![Screenshot at 2021-03-15 22-21-52](https://user-images.githubusercontent.com/3116038/111206027-20d48f80-85dd-11eb-836e-f8c7095ec507.png)
